dashboard/app/models/levels/standalone_video2.rb

```diff
@@ -0,0 +1,77 @@
+# == Schema Information
+#
+# Table name: levels
+#
+#  id                    :integer          not null, primary key
+#  game_id               :integer
+#  name                  :string(255)      not null
+#  created_at            :datetime
+#  updated_at            :datetime
+#  level_num             :string(255)
+#  ideal_level_source_id :bigint           unsigned
+#  user_id               :integer
+#  properties            :text(4294967295)
+#  type                  :string(255)
+#  md5                   :string(255)
+#  published             :boolean          default(FALSE), not null
+#  notes                 :text(65535)
+#  audit_log             :text(65535)
+#
+# Indexes
+#
+#  index_levels_on_game_id    (game_id)
+#  index_levels_on_level_num  (level_num)
+#  index_levels_on_name       (name)
+#
+
+# StandaloneVideo2 uses level_data, which is actual JSON in each .level file.
+
+class StandaloneVideo2 < Level
```

Comment:
should `StandaloneVideo2` extend from `StandaloneVideo`?

Reply:
I think the server-side model should be different between them, since `StandaloneVideo2` would start to rely on a `level_data` JSON blob rather than the separate serialized attributes used in `StandaloneVideo`, and probably doesn't need to inherit all of the older model's methods.

Reply:
As mentioned elsewhere, this change no longer adds a new model on the server, and instead just adds a new client for standalone video levels that are shown in a `LabContainer` context.